src/Microsoft.AspNet.Razor/CodeGenerators/CSharpTagHelperCodeRenderer.cs

```diff
@@ -708,7 +711,7 @@ private void RenderBufferedAttributeValueAccessor(CSharpCodeWriter writer)
                     StringValueBufferVariableName,
                     _tagHelperContext.TagHelperContentGetContentMethodName,
                     endLine: false,
-                    parameters: new string[] { _tagHelperContext.HtmlEncoderPropertyName });
+                    parameters: _tagHelperContext.HtmlEncoderPropertyName);
```

Comment:
Minor nit I fixed in passing. Array is allocated regardless but this way is easier on the eyes.